main/acle.md

```diff
@@ -1648,6 +1650,11 @@ be found in [[BA]](#BA).
 | `__ARM_SIZEOF_WCHAR_T` [[ssec-Imptype]](#implementation-defined-type-properties)                                         | Size of `wchar_t`: 2 or 4                                                                          | 2           |
 | `__ARM_WMMX` [[ssec-WMMX]](#wireless-mmx)                                                         | Wireless MMX extension (32-bit-only)                                                               | 1           |
 
+## NEON-SVE Bridge macros
+
+``__ARM_NEON_SVE_BRIDGE`` is defined to 1 if NEON-SVE Bridge 
```

Comment:
Please use single quotes for fixed-font text (all through the patch, I will not mark the other places). It also seems that there is a duplicate word?

```suggestion
`__ARM_NEON_SVE_BRIDGE` is defined to 1 if
```